ghost/core/core/server/api/endpoints/posts.js

```diff
@@ -279,5 +279,30 @@ module.exports = {
         query(frame) {
             return models.Post.destroy({...frame.options, require: true});
         }
+    },
+
+    copy: {
+        statusCode: 201,
+        headers: {},
+        options: [
+            'id',
+            'formats'
+        ],
+        validation: {
+            id: {
+                required: true
+            }
+        },
+        permissions: {
+            method: 'add'
+        },
+        async query(frame) {
+            return postsService.copyPost(frame)
+                .then((model) => {
+                    this.headers.cacheInvalidate = false;
```

Comment:
same thing as [here](https://github.com/TryGhost/Ghost/pull/16767/files#r1190778503)

Reply:
Fixed in https://github.com/TryGhost/Ghost/pull/16767/commits/bd8dc9037be19589336dd0bb834ab58ccfe63206